src/esp/metadata/attributes/ObjectAttributes.cpp

```diff
@@ -12,6 +12,8 @@ AbstractObjectAttributes::AbstractObjectAttributes(
     const std::string& handle)
     : AbstractAttributes(attributesClassKey, handle) {
   setFrictionCoefficient(0.5);
+  setRollingFrictionCoefficient(0.0);
```

Comment:
We plan to find the correct coefficients through experiments?

Reply:
0 was chosen for the default for backwards compatibility (to mirror the current Bullet defaults). 
We'll set this on a case by case basis.